themes/default/content/_index.md

```diff
@@ -40,95 +40,257 @@ overview:
       - /logos/tech/ci-cd/travis-ci.svg
 
 build:
-  title: Build
+  title: Write your configuration quickly
   description: |
-    Build cloud applications and infrastructure by combining the safety and reliability of [infrastructure as code](/what-is/what-is-infrastructure-as-code/) with the power of familiar programming languages and tools.
+    Pulumi helps you speed up your inner dev loop for IaC by allowing you to use an IDE giving you statement completion, real-time type checking, and interactive documentation working out the box. In addition you can leverage all the capabilities of the programming language of your choice to help you reduce the total number of lines of code you're writing.
   items:
     - title: Flexible and expressive
-      description: Loops, conditionals, functions, classes, and more.
-
-    - title: Be productive
-      description: Gets things done in seconds rather than hours.
+      description: |
+        Pulumi allows you to write expressive and easily legible logic into your configurations, removing the need for clever workarounds to handle simple use cases.
 
     - title: Share and reuse
-      description: Define and consume patterns and practices to reduce boilerplate.
-
-use_cases:
-  - title: Write your configuration quickly
-    icon: code
-    icon_color: purple
-    description: |
-      Speed up your inner dev loop with statement completion, real-time type checking, and interactive documentation directly in your IDE.
-
-  - title: Ship your applications faster
-    icon: guage
-    icon_color: yellow
-    description: |
-      Integrate Pulumi directly with your CI/CD provider to deliver your applications and infrastructure with more confidence and transparency.
-
-  - title: Doing more with less
-    icon: rocketship
-    icon_color: blue
-    description: |
-      You can use the [Automation API](/automation/) to create tooling that help you and your engineers manage **10x the amount of resources** versus traditional tooling.
+      description: |
+        You can use Pulumi Packages to build best-practice abstractions available in all supported languages.
```

Comment:
I think we should remove this section and then build it into the new Ship applications faster section